src/components/Item.js

```diff
@@ -5,24 +5,36 @@ import React from 'react';
 import richTextHelper from '../shared/rich-text';
 import SaveButton from './SaveButton';
 
-const Item = ({ item, canBeSaved }) => (
-  <Box
-    display="flex"
-    flexDirection="column"
-    direction="column"
-    pt={3.5}
-    px={2}
-  >
-    <Typography variant="h1" align="center">{item.title}</Typography>
-
-    {canBeSaved ? <SaveButton resourceSlug={item.slug} /> : null}
-
-    {item.type === 'external_link'
-      ? <LinkUi href={item.link} color="inherit">Go to resource</LinkUi>
-      : richTextHelper(item.content)}
-
-  </Box>
-);
+const Item = ({ item, canBeSaved }) => {
+  const richTextTransformer = (node) => {
+    if (node.type === 'tag' && node.name === 'p' && node.children[0] && node.children[0].name === 'img') {
```

Comment:
Is it possible to just do `node.type === 'tag' && node.name === 'img'` here to catch _all_ images in the rich text (otherwise in this case it only catches images that are the first child within a `<p>` tag).

Reply:
That was my first thought too, but if an image is on a new line all by itself it will render inside a <p> and that's what's causing it to overflow.

When you include an image on the storybook rich text editor it always puts in on its own line, anyway so this should capture all the images anyway!

Reply:
> […] if an image is on a new line all by itself it will render inside a […]

Is that the case if the image is added to a list, or link (or maybe even a table [if we add that later])? Not sure if these are relevant use cases; your solution does solve the primary use case for now, but if it's an easy fix then it seems to make sense to do so now.